Add "Upgrading installed Operators" (#26272)
openshift/openshift-docs: updating/updating-cluster-cli.adoc

* Have access to the cluster as a user with `admin` privileges.
See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
* Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].

> * Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) have been updated to their latest version in their latest channel. This ensures a valid upgrade path for each Operator when the default OperatorHub catalogs switch from the current minor version to the next during the cluster upgrade. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.

Reviewer: ```suggestion
* Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures a valid upgrade path for each Operator when the default OperatorHub catalogs switch from the current minor version to the next during the cluster upgrade. See xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators] for more information.
```
